src/config/webpack.config.js

```diff
@@ -92,14 +94,14 @@ const defaultWebpackConfig = {
       template: path.join(__dirname, '..', 'index.html'),
       chunks: ['babel-polyfill', 'terra-site'],
     }),
-    new webpack.DefinePlugin({
-      SITE_CONFIG: JSON.stringify(siteConfig),
-      COMPONENT_CONFIG_PATH: JSON.stringify(componentConfigPath),
-    }),
     new I18nAggregatorPlugin({
       baseDirectory: process.cwd(),
       supportedLocales: i18nSupportedLocales,
     }),
+    new webpack.DefinePlugin({
```

Comment:
Moved after I18nPlugin so I don't break the temp i18n fix int he consumer-theme repo.